include/content.js

```diff
@@ -26,7 +26,10 @@ module.exports = function(pb) {
      * @class ContentService
      * @constructor
      */
-    function ContentService(){}
+    function ContentService(site, onlyThisSite) {
+        this.siteUid = pb.SiteService.getCurrentSite(site);
+        this.settingService = pb.SettingServiceFactory.getServiceBySite(this.siteUid, onlyThisSite);
+    }
```

Comment:
For constructors, instead of multiple arguments we should pass an object.  This allows us to easily expand options without have to worry so much about breaking changes down the road.